src/main/java/org/opensearch/security/dlic/rest/api/AuthTokenProcessorAction.java

```diff
@@ -58,8 +59,8 @@ public AuthTokenProcessorAction(final Settings settings, final Path configPath,
 	}
 
 	@Override
-	public List<Route> routes() {
-		return routes;
+	public List<ReplacedRoute> replacedRoutes() {
+	    return replacedRoutes;
```

Comment:
this line uses both spaces and tabs, it must use tabs for indentation (please check other files for the same)

Reply:
Fixed it in `AuthTokenProcessorAction` , `PermissionsInfoAction` and  `RolesApiAction`, other files does not have this issue.

Reply:
There are still more files with the issue.